src/wallet/wallet.cpp

```diff
@@ -1420,6 +1420,37 @@ void CWallet::transactionAddedToMempool(const CTransactionRef& tx) {
     if (it != mapWallet.end()) {
         RefreshMempoolStatus(it->second, chain());
     }
+
+    const Txid& conflict_txid = tx->GetHash();
+
+    for (const CTxIn& tx_in : tx->vin) {
+        // If the wallet contains no transactions that spend the same prevout, continue
+        if (mapTxSpends.count(tx_in.prevout) == 0) continue;
+
+        std::pair<TxSpends::const_iterator, TxSpends::const_iterator> range;
+        range = mapTxSpends.equal_range(tx_in.prevout);
+
+        // For each wallet transaction spending this prevout...
+        for (TxSpends::const_iterator _it = range.first; _it != range.second; ++_it) {
```

Comment:
In commit "wallet: track mempool conflicts" (2c867784b2a719c9e701e19e389c3adfa7d7faae)

Just IMO, but

```c++
auto range =  mapTxSpends.equal_range(tx_in.prevout);
for (auto it = range.first; it != range.second; ++it)
```

would make core more readable, dropping the long type declarations and extra underscore. But feel free to keep current style if that is your preference.

Reply:
I have implemented most of this, except for removing the extra underscore, because `it` was already defined above.